material/30days-curriculum/day02_ダッシュボードに自分だけのメッセージを追加しよう.md

```
@@ -1199,23 +1272,68 @@ export default function DashboardPage() {
 
 - 自分専用の画面づくりは、「誰の画面で、今日は何に集中するのか」が見えるようにするところから始まる
 - 値はその場に直接書き散らすより、型（`type`）や関数にまとめて意味を持たせたほうが、あとから変更しやすい
 - クリックなどの操作がない画面は、無理に Client Component にせず、Server Component のままにしておく
 
-この3つが入るだけで、
-Day 02 のコードはかなり「プロダクトを育てる書き方」に近づきます。
-## 明日のプレビュー
+この3つが入っていれば、名前や集中テーマを変えたくなったときに
+触るのはファイル先頭の1行だけで、
+`{ownerName}`・`{focusTheme}`・`{todayNote}` の3か所が同時に変わります。
+## つまずきポイント
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)